OneCommunityGlobal/HighestGoodNetworkApp
# Anirudh -  AddTeamsAutoComplete unit test

# Description

Unit tests for src/components/UserProfile/TeamsAndProjects/AddTeamsAutoComplete.jsx

## Main changes explained:

New File
- AddTeamsAutoComplete.test.js : Unit test case file

Test Cases:

![Screenshot 2024-02-02 at 8 01 24 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/e2bf6e67-2500-4883-83e0-2af8d443613e)


## How to test:
check into current branch
do npm test AddTeamsAutoComplete.test.js 
check if all the test cases pass

## Screenshots or videos of changes:

![AddTeamsAutoComplete - UnitTest working](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/46c3f8aa-0c39-49ad-9506-b3c98c4a0032)


## Note:
Include the information the reviewers need to know.
